src/Microsoft.VisualStudio.Composition.LocalizationShell/MultilingualResources/Microsoft.VisualStudio.Composition.LocalizationShell.cs.xlf

```diff
@@ -112,12 +113,16 @@
           <target state="translated">Ke každému importu musí být právě jedna položka.</target>
         </trans-unit>
         <trans-unit id="ExpectedExactlyOneExportButFound" translate="yes" xml:space="preserve">
-          <source>{0}: expected exactly 1 export of {1} but found {2}.{3}</source>
-          <target state="translated">{0}: očekával se právě 1 export {1}, ale počet nalezených exportů je: {2}.{3}</target>
+          <source>{0}: expected exactly 1 export matching constraints:
+{1}but found {2}.{3}</source>
```

Comment:
I think we lost a space here between '{0}' and 'but'

Reply:
This is intentional. `{1}` gets replaced with a series of lines that end in a newline character, so the space that _so wants to be there_ would actually cause ` but found` to appear to have an extra space at the start of its new line.